ui/job-view/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
+
+// Vendor Styles
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'font-awesome/css/font-awesome.css';
+
+// Vendor JS
+import 'bootstrap';
+import 'jquery.scrollto';
+
+// Treeherder Styles
+import '../css/treeherder.css';
+import '../css/treeherder-global.css';
+import '../css/treeherder-navbar.css';
+import '../css/treeherder-navbar-panels.css';
+import '../css/treeherder-notifications.css';
+import '../css/treeherder-details-panel.css';
+import '../css/treeherder-job-buttons.css';
+import '../css/treeherder-resultsets.css';
+import '../css/treeherder-pinboard.css';
+import '../css/treeherder-bugfiler.css';
+import '../css/treeherder-loading-overlay.css';
+
+import JobView from './JobView';
```

Comment:
We could always rename `JobView.jsx` to `App.jsx` if we wanted, for consistency with the other pages? (I agree with Sarah that it's good to not have both). I guess it would also make it easier for someone browsing the source to see where the main React component lived?

Though I'm happy either way, so up to you :-)

Reply:
Yeah, I went back and forth on this a couple times before requesting review.  Thank you both for your opinions.  I think I *DO* like having ``App`` be the main entry point.  It sort of feels like ``JobView`` could be broken up to ``App`` and ``JobView``.  Probably where ``App`` has all the Contexts, and ``JobView`` starts with ``PrimaryNavBar``.  I don't feel like that's appropriate to split that up in this PR.  But I may experiment with that later.  It would help if we add more Contexts (like an Auth context, or whatever) that it won't affect so much component nesting in ``render()``.

Reply:
Makes sense! In which case we could just leave as-is if splitting up further later, to avoid git rename churn?